pjlib-util/src/pjlib-util-test/http_client.c

```diff
@@ -116,14 +116,24 @@ static int server_thread(void *p)
         } else if (srv->action == ACTION_REPLY) {
             pj_size_t send_len = 0;
             unsigned ctr = 0;
-            pj_ansi_sprintf(pkt, "HTTP/1.0 200 OK\r\n");
+            pkt_len = pj_ansi_snprintf(pkt, srv->buf_size, 
+                                       "HTTP/1.0 200 OK\r\n");
+            PJ_ASSERT_ON_FAIL(pkt_len>0, {
+                PJ_PERROR(2, (THIS_FILE, -pkt_len, "Error creating response"));
+                pj_sock_close(newsock);
+                continue;
+            })
             if (srv->send_content_length) {
-                pj_ansi_sprintf(pkt + pj_ansi_strlen(pkt), 
+                pj_ansi_snprintf(pkt + pkt_len, srv->buf_size - pkt_len,
                                 "Content-Length: %d\r\n",
                                 srv->data_size);
```

Comment:
a code indent  problem.  Better to check all `sprintf`  change to `snprintf`

Reply:
intentional to minimize code change (re: git blame)